Userland/Libraries/LibWeb/Platform/EventLoopPluginSerenity.h

```diff
@@ -6,9 +6,9 @@
 
 #pragma once
 
-#include <LibWeb/Platform/EventLoopPlugin.h>
+#include "EventLoopPlugin.h"
```

Comment:
y tho, this is not our usual style for serenity libs

Reply:
Oops! I got CLion to move the files, I thought I fixed all the includes after but I obviously missed this one.